client-v2/src/components/FrontsCAPIInterface/Feed.tsx

```diff
@@ -29,25 +25,7 @@ const Feed = ({ articles = [], error }: FeedProps) => (
     {articles.length ? (
       articles
         .filter(result => result.webTitle)
-        .map(article => (
-          <FeedItem
-            id={article.id}
-            key={article.webUrl}
-            title={article.webTitle}
-            href={article.webUrl}
-            publicationDate={article.webPublicationDate}
-            sectionName={article.sectionName}
-            pillarId={article.pillarId}
-            internalPageCode={
-              article.fields && getId(article.fields.internalPageCode)
-            }
-            firstPublicationDate={article.fields.firstPublicationDate}
-            isLive={!article.fields.isLive || article.fields.isLive === 'true'}
-            scheduledPublicationDate={article.fields.scheduledPublicationDate}
-            thumbnail={getThumbnail(article, article.frontsMeta.defaults)}
-            tone={article.frontsMeta.tone}
-          />
-        ))
+        .map(article => <FeedItem key={article.id} article={article} />)
```

Comment:
Heh. Just for the sake of cleanliness, or were there optimisation gains to be had here? (Possibly getThumbnail?)

Reply:
Ah as above really, just needed the whole article in there for serialisation so decided it would be cleaner to remove the props that were just derived from an article.

Reply:
👍